.github/workflows/tests.yml

```diff
@@ -295,10 +295,41 @@ jobs:
           status: ${{ job.status }}
           channel: '#haystack'
         if: failure() && github.repository_owner == 'deepset-ai' && github.ref == 'refs/heads/main'
+
+  integration-tests-faiss:
+    name: Integration / faiss / ${{ matrix.os }}
+    needs:
+     - unit-tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest,macos-latest,windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: ./.github/actions/python_cache/
+
+      - name: Install Haystack
+        run: pip install -U .
```

Comment:
`-U` is [problematic for `hatch-autorun`](https://github.com/deepset-ai/haystack/issues/3505), do we really need it? Let's make sure it doesn't cause problems

Reply:
I thought that was a requirement? https://github.com/deepset-ai/haystack/blob/43b24fd1a79677d070b03ce1c5c1e122f0b9ea69/.github/actions/python_cache/action.yml#L45-L48

Reply:
Oh right, my bad :smile: the schema should be generated by the action, you're right.